scrypto/src/component/component.rs

```diff
@@ -208,8 +208,6 @@ pub enum ComponentAddress {
     System([u8; 26]),
 }
 
-impl ComponentAddress {}
```

Comment:
those empty impl blocks were shadowing the impl below (with `to_vec`), not sure if that's expected in rust, or a bug (or I didn't know how to use it)

Reply:
It should no shadow other `impl` for the struct, but we should remove it.

Reply:
It should not shadow other `impl`s, but it's nice to remove them.